src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaManagerDataStore.cs

```diff
@@ -123,23 +116,29 @@ public async Task ExecuteScriptAsync(string script, CancellationToken cancellati
 
         /// <inheritdoc />
         public async Task<bool> BaseSchemaExistsAsync(CancellationToken cancellationToken)
+        {
+            return await ObjectExistsAsync("SelectCurrentVersionsInformation", "P", cancellationToken);
```

Comment:
nit: no need to `await` here. Simply pass the task directly:
```suggestion
            return ObjectExistsAsync("SelectCurrentVersionsInformation", "P", cancellationToken);
```

Reply:
umm, it expects return type as bool rather than  Task<bool>. I guess I am not following :(